File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        insert_index = 1
        for ind in range(1, len(nums)):
            if nums[ind-1] != nums[ind]:
                nums[insert_index] = nums[ind]
                insert_index += 1
        return insert_index

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int insertIndex = 1;
        for(int ind=1; ind < nums.length; ind++){
            if(nums[ind-1] != nums[ind]){
                nums[insertIndex] = nums[ind];
                insertIndex ++;
            }
        }
        return insertIndex;
    }
}